init.ps1

```diff
@@ -1,57 +0,0 @@
-<#
```

Comment:
correct. Instead, we run `msbuild /t:restore` or just let VS2017 do it automatically if you're in the IDE.